app/routes/__index/domains/index.tsx

```diff
@@ -1,13 +1,18 @@
 import { AddIcon } from '@chakra-ui/icons';
-import { Button, Container, Flex, Heading, Text } from '@chakra-ui/react';
+import { Button, Container, Flex, Heading, Icon, IconButton, Text } from '@chakra-ui/react';
```

Comment:
```suggestion
import { Button, Container, Flex, Heading, Icon, Text } from '@chakra-ui/react';
```

remove the button if you aren't going to use it